src/api/services/publish.js

```diff
@@ -9,6 +8,8 @@ import {
     SCOPE_GRAPHIC,
 } from '../../common/scope';
 
+const isJobActive = async ctx => !ctx.job || (await ctx.job.isActive());
```

Comment:
This function either returns a boolean or a promise, isn't it a code smell to have 2 different return types?